graphql_compiler/compiler/filters.py

```diff
@@ -543,6 +543,86 @@ def _process_has_substring_filter_directive(filter_operation_info, location, con
     return blocks.Filter(filter_predicate)
 
 
+@scalar_leaf_only(u'ends_with')
+@takes_parameters(1)
+def _process_ends_with_filter_directive(filter_operation_info, location, context, parameters):
+    """Return a Filter basic block that checks if the directive arg is the string suffix of the field.
+    Args:
```

Comment:
Use a newline between sections. The docstring sections should be like this:

```python
"""Headline, always one line

[Optional additional text, as many lines as you'd like, same indentation as the triple-quotes]

Args:
    foo: explanation
    bar: explanation

Returns: [only if the function returns something]
    return value description (no name, since return values aren't named)

Raises: [only if the function raises something non-obvious]
    description of errors raised
"""
```

This is missing newlines before `Args` and `Returns`.